tests/unit/SidebarTest.ts

```diff
@@ -70,36 +60,48 @@ describe('Sidebar', () => {
             return (
                 waitForBatchedUpdates()
                     // When Onyx is updated with the data and the sidebar re-renders
-                    .then(() =>
-                        Onyx.multiSet({
+                    .then(() => {
+                        const reportCollection = {
```

Comment:
You should also be able to reuse CollectionDataSet based type here. See the [example](https://github.com/Expensify/App/pull/37204/files#diff-3dbd2ccb0d40c7436a46592db38b5c76a72ad90def10cea969a037d6d7480658R93) from my PR (there are even several cases like that)